linera-execution/tests/wasm.rs

```diff
@@ -24,10 +24,10 @@ use test_case::test_case;
 /// called correctly and consume the expected amount of fuel.
 ///
 /// To update the bytecode files, run `linera-execution/update_wasm_fixtures.sh`.
-#[cfg_attr(with_wasmer, test_case(WasmRuntime::Wasmer, 90_090; "wasmer"))]
-#[cfg_attr(with_wasmer, test_case(WasmRuntime::WasmerWithSanitizer, 90_622; "wasmer_with_sanitizer"))]
-#[cfg_attr(with_wasmtime, test_case(WasmRuntime::Wasmtime, 90_454; "wasmtime"))]
-#[cfg_attr(with_wasmtime, test_case(WasmRuntime::WasmtimeWithSanitizer, 90_454; "wasmtime_with_sanitizer"))]
+#[cfg_attr(with_wasmer, test_case(WasmRuntime::Wasmer, 92357; "wasmer"))]
+#[cfg_attr(with_wasmer, test_case(WasmRuntime::WasmerWithSanitizer, 92917; "wasmer_with_sanitizer"))]
+#[cfg_attr(with_wasmtime, test_case(WasmRuntime::Wasmtime, 92749; "wasmtime"))]
+#[cfg_attr(with_wasmtime, test_case(WasmRuntime::WasmtimeWithSanitizer, 92749; "wasmtime_with_sanitizer"))]
```

Comment:
Nit: adding an underscore to make it easier to read.

```suggestion
#[cfg_attr(with_wasmer, test_case(WasmRuntime::Wasmer, 92_357; "wasmer"))]
#[cfg_attr(with_wasmer, test_case(WasmRuntime::WasmerWithSanitizer, 92_917; "wasmer_with_sanitizer"))]
#[cfg_attr(with_wasmtime, test_case(WasmRuntime::Wasmtime, 92_749; "wasmtime"))]
#[cfg_attr(with_wasmtime, test_case(WasmRuntime::WasmtimeWithSanitizer, 92_749; "wasmtime_with_sanitizer"))]
```